bioblend/galaxy/jobs/__init__.py

```diff
@@ -70,6 +70,48 @@ def show_job(self, job_id, full_details=False):
 
         return self._get(id=job_id, params=params)
 
+    def build_for_rerun(self, job_id):
+        """
+        Get details of a given job of the current user.
+
+        :type job_id: str
+        :param job_id: job ID
+
+        :rtype: dict
+        :return: A description of the given job, with all parameters required to rerun.
+
+        """
+        url = '/'.join((self._make_url(job_id), 'build_for_rerun'))
+        return self._get(url=url)
+
+    def remap_job(self, job_id):
+        """
+        Rerun a failed job, remapping outputs so that the failed dataset is replaced
+        with the new output.
+
+        This is a convenience bioblend method which combines two API calls - the first
+        to get the input parameters for the job, the second to rerun it.
+
```

Comment:
Unnecessary details IMO.

```suggestion
        If other tools were waiting for this job to finish successfully, this
        will resume those tools using the new outputs of this tool run.
```